### PR TITLE
Misc cleanup before a release

### PR DESCRIPTION
**Pull request by tas50**
Sync up the cookbook with the format / testing process of our other cookbooks

### CODE REVIEW

**Comment by tas50** [comment]
Ping @mtyler @tfitch @yzl for review

File: libraries/_helper.rb [hunk 1]
```diff
@@ -47,7 +47,7 @@ def initialize(endpoint, timeout)
 
     # Matches Version 4 UUID per RFC 4122
     # Example: 38537014-ec66-49b5-aff2-aed1c19e2989
-    UUID_REGEX = /[a-f0-9]{8}-[a-f0-9]{4}-4[a-f0-9]{3}-[89aAbB][a-f0-9]{3}-[a-f0-9]{12}/.freeze unless defined?(UUID_REGEX)
+    UUID_REGEX = /[a-f0-9]{8}-[a-f0-9]{4}-4[a-f0-9]{3}-[89aAbB][a-f0-9]{3}-[a-f0-9]{12}/ unless defined?(UUID_REGEX)
```
Comment (yzl) [inline libraries/_helper.rb]: I thought ruby constants were mutable, in which case it makes sense to me to use `freeze` here.

Comment (tas50) [inline libraries/_helper.rb]: Rubocop seems to think that's not necessary there

**Comment by tas50** [comment]
@yzl @schisamo @tfitch @mtyler Can I get a review on this PR?

I'd like to get the current changes on master + these docs changes out.  This is failing on the same errors that are currently in master.

**Pull request merged**